addons/l10n_id_efaktur/models/efaktur_document.py

```diff
@@ -0,0 +1,19 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class EfakturDocument(models.Model):
+    _name = "l10n_id_efaktur.document"
+    _description = "Managing the E-faktur attachment on invoice and download"
```

Comment:
```suggestion
    _description = "E-faktur Document"
```
I believe we usually use these to describe what this model is about, not what it does.
The ORM says: `#: the model's informal name`